AppBidding-MoPub/AppBidding-MoPub/AdConfigurations.h

```diff
@@ -0,0 +1,28 @@
+//
+//  AdConfigurations.h
+//  AppBidding-MoPub
+//
+//  Created by Abdoul Guisset.
+//  Copyright © 2020 Abdoul. All rights reserved.
```

Comment:
Let's keep file-level attribution generic to Criteo (multiple occurences in this PR)

```suggestion
//  Copyright © 2020 Criteo. All rights reserved.
```

Reply:
+1

The LICENSE file at the root of this project shows that this preamble shoudld be added at the top of all files:

```
//  Copyright © 2018-2020 Criteo. All rights reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
// http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Reply:
Good point, will update it.